doc/strace.1.in

```diff
@@ -1010,6 +1011,10 @@ Print the syscall number.
 .if '@USE_LIBDW_FALSE@'#' Print the execution stack trace and source code information of the traced
 .if '@USE_LIBDW_FALSE@'#' processes after each system call. This option expects the target program is compiled
 .if '@USE_LIBDW_FALSE@'#' with appropriate debug options: "\-g" (gcc), or "\-g \-gdwarf-aranges" (clang).
+.if '@ENABLE_STACKTRACE_FALSE@'#' .TP
+.if '@ENABLE_STACKTRACE_FALSE@'#' .B \-\-stacktrace\-frames\-limit [= limit ]
+.if '@ENABLE_STACKTRACE_FALSE@'#' Print no more than this amount of stack trace frames
+.if '@ENABLE_STACKTRACE_FALSE@'#' when backtracing a system call.
```

Comment:
How about adding 
```
Use with 
.B \-\-stack\-traces
(or 
-B \-k
) options. 
```
?

Reply:
I'd assume this is implied, but you are right, being more explicit wouldn't hurt.
Done